docs/ui-components/price-breakdown.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Price Breakdown
+parent: UI Components
+nav_order: 4
+---
+
+# Price Breakdown
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+The price breakdown component displays information about Afterpay instalments and handles a number of common configurations.
+
+A total payment amount (represented as a `BigDecimal`) must be programatically set on the component to display Afterpay instalment information.
+
+```kotlin
+val totalAmount: BigDecimal = getTotalAmount()
+
+val paymentBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.priceBreakdown)
+paymentBreakdown.totalAmount = totalAmount
+```
+
+When the breakdown component is assigned a total amount that is valid for the merchant account, the 4 instalment amounts will be displayed.
+
+![Price breakdown Afterpay instalments are available][breakdown-available]
+
+When the total amount is not within the minimum and maximum payment values for the merchant account, the amounts available for Afterpay will be shown in the component.
+
+![Price breakdown Afterpay instalments are unavailable][breakdown-unavailable-min-max]
+
+When no minimum amount is set and the total amount is greater than the maximum payment values for the merchant account, the maximum amount available for Afterpay will be shown in the component.
+
+![Price breakdown Afterpay instalments are unavailable][breakdown-unavailable-max]
+
+When no payment amount has been set or the merchant account configuration has not been applied to the SDK, the component will default to a message stating Afterpay is available.
+
+![Price breakdown no merchant account configuration][breakdown-no-configuration]
+
+The **Info** link at the end of the component will display a window containing more information about Afterpay for the user.
+
+## Configuring the Price Breakdown
+{: .no_toc }
+
+### Intro Text
+Setting `introText` is optional, will default to `OR` and must be of type `AfterpayIntroText`.
+
+Can be any of `OR`, `OR_TITLE`, `MAKE`, `MAKE_TITLE`, `PAY`, `PAY_TITLE`, `IN`, `IN_TITLE`, `PAY_IN`, `PAY_IN_TITLE` or `EMPTY` (no intro text).
+Intro text will be rendered lowercase unless using an option suffixed with `_TITLE` in which case title case will be rendered.
```

Comment:
It's obviously too late to provide this feedback, but I've just realised we should have used `_SENTENCE` instead of `_TITLE` as a suffix for capitalising the first letter. "Pay in" is sentence case, whereas "Pay In" is title case.